packages/com.influxdata.telegraf/plugins/inputs/Jolokia2AgentInput.pkl

```diff
@@ -0,0 +1,87 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+/// [Jolokia Agent Input Plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/jolokia2_agent)
+///
+/// Reads JMX metrics from one or more Jolokia agent REST endpoints.
+/// See [docs](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/jolokia2_agent/README.md) for examples.
+@ModuleInfo { minPklVersion = "0.24.0" }
```

Comment:
This version precedes the first Pkl version. But also, this happens to be compatible with all Pkl versions, so, I don't see a particular need to add this.

```suggestion
```